src/ports/postgres/modules/lda/lda.sql_in

```diff
@@ -182,324 +105,789 @@ lda_train( data_table,
+    <dd>TEXT. The name of the table generated by LDA that stores 
+    the output data. It has the following columns:
         <table class="output">
             <tr>
                 <th>docid</th>
-                <td>INTEGER.</td>
+                <td>INTEGER. Document id from input 'data_table'.</td>
             </tr>
             <tr>
                 <th>wordcount</th>
-                <td>INTEGER.</td>
+                <td>INTEGER. Count of number of words in the document, 
+                including repeats. For example, if a word appears 3 times 
+                in the document, it is counted 3 times.</td>
             </tr>
             <tr>
                 <th>words</th>
-                <td>INTEGER[].</td>
+                <td>INTEGER[]. Array of \c wordid in the document, not
+                including repeats.  For example, if a word appears 3 times 
+                in the document, it appears only once in the \c words array.</td>
             </tr>
             <tr>
                 <th>counts</th>
-                <td>INTEGER[].</td>
+                <td>INTEGER[]. Frequency of occurance of a word in the document,
+                indexed the same as the \c words array above.  For example, if the
+                2nd element of the \c counts array is 4, it means that the word
+                in the 2nd element of the \c words array occurs 4 times in the
+                document.</td>
             </tr>
             <tr>
                 <th>topic_count</th>
-                <td>INTEGER[].</td>
+                <td>INTEGER[]. Array of the count of words in the document
+                that correspond to each topic.</td>
             </tr>
             <tr>
                 <th>topic_assignment</th>
-                <td>INTEGER[].</td>
+                <td>INTEGER[]. Array indicating which topic each word in the 
+                document corresponds to.  This array is of length \c  wordcount.</td>
             </tr>
         </table>
     </dd>
     <dt>voc_size</dt>
-    <dd>INTEGER. Size of the vocabulary. Note that the \c wordid should be continous integers starting from 0 to \c voc_size &minus; \c 1.  A data validation routine is called to validate the dataset.</dd>
+    <dd>INTEGER. Size of the vocabulary. As mentioned above for the 
+                input 'data_table', \c wordid consists of continous integers going 
+                from 0 to \c voc_size &minus; \c 1.   
+    </dd>
     <dt>topic_num</dt>
-    <dd>INTEGER. Number of topics.</dd>
+    <dd>INTEGER. Desired number of topics.</dd>
     <dt>iter_num</dt>
-    <dd>INTEGER. Number of iterations (e.g. 60).</dd>
+    <dd>INTEGER. Desired number of iterations.</dd>
     <dt>alpha</dt>
-    <dd>DOUBLE PRECISION. Dirichlet parameter for the per-doc topic multinomial (e.g. 50/topic_num).</dd>
+    <dd>DOUBLE PRECISION. Dirichlet prior for the per-document topic 
+    multinomial (e.g., 50/topic_num is a typical value to start with).</dd>
```

Comment:
I found different libraries do use different starting value, e.g. 1/k, 5/k and 0.1. We can mention this value (50/k) is suggested in Griffiths and Steyvers Paper.